### DISCUSSION TITLE
Can we limit the ABR with MediaMTX ?

### DISCUSSION THREAD

**Discussion by huxaiphaer** [bluenviron/mediamtx#3451] [General]
I am try to limit the ABR to only 480p, how can I do this ?  I am trying to make the livestream more smoother, I am already uisng a CDN of bunny.net plus a VM of Azure (16 GB RAM), but still the stream is freezing somehow.

**Comment by aler9** [comment]
Hello, MediaMTX doesn't touch the stream bitrate or content in any way, it's the user responsibility to generate a stream that is fit to your specific needs. If you really want to perform a server side re-encoding in order to change the bitrate, instructions are in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#remuxing-re-encoding-compression

[a comment by an automated account is not shown]
